x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/processor_form/processors/csv.tsx

```diff
@@ -148,17 +149,41 @@ export const CSV: FunctionComponent = () => {
         config={fieldsConfig.target_fields}
         component={ComboBoxField}
         path="fields.target_fields"
+        data-test-subj="targetFieldsField"
       />
 
-      <UseField config={fieldsConfig.separator} component={Field} path="fields.separator" />
+      <UseField
+        config={fieldsConfig.separator}
+        component={Field}
+        path="fields.separator"
+        data-test-subj="separatorValueField"
+      />
 
-      <UseField config={fieldsConfig.quote} component={Field} path="fields.quote" />
+      <UseField
+        config={fieldsConfig.quote}
+        component={Field}
+        path="fields.quote"
+        data-test-subj="quoteValueField"
+      />
 
-      <UseField config={fieldsConfig.trim} component={ToggleField} path="fields.trim" />
+      <UseField
+        config={fieldsConfig.trim}
+        component={ToggleField}
+        path="fields.trim"
+        data-test-subj="trimSwitch"
+      />
 
-      <UseField config={fieldsConfig.empty_value} component={Field} path="fields.empty_value" />
+      <UseField
+        config={fieldsConfig.empty_value}
+        component={Field}
+        path="fields.empty_value"
+        data-test-subj="emptyValueField"
+      />
 
-      <IgnoreMissingField />
+      <IgnoreMissingField
+        defaultValue={true}
```

Comment:
I think the change you wanted to do is inside the `x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/processor_form/processors/common_fields/ignore_missing_field.tsx` file on L25

Reply:
Sorry, I missed the context. The default value depends on the context (it is only `true` for CSV processor right?)

Reply:
Yes, you are right on point! So far it only applies to `registered_domain` and this `csv` one. The rest of the processors share the same default behaviour stated by the `IgnoreMissingField` component.